lib/onlyoffice_webdriver_wrapper/helpers/chrome_helper.rb

```diff
@@ -37,6 +37,7 @@ def start_chrome_driver
       switches = add_useragent_to_switches(DEFAULT_CHROME_SWITCHES)
       options = Selenium::WebDriver::Chrome::Options.new(args: switches,
                                                          prefs: prefs)
+      options.add_argument("--unsafely-treat-insecure-origin-as-secure=#{ENV['PortalIP']}") if ENV['PortalIP']
```

Comment:
@l8556 I would give two advices about this MR 

1. Small note - In CHANGELOG it's always a good idea to specify `not what changed` but `how can I use it`
So the line would be `Added PortalIP env var to allow download file via HTTP without confirmation`
Because the whole Chrome flag name is more like an internal implementation

2. More important one - when adding options like this try to name ENV var specific to current library, not specific to project which would use the library
Like if I would use this lib for some project like `api.temlab.info` - what it's `PortalIP` in context of api.teamlab.info - we don't have any Portal, it's just a site we're testing

I think better name (considering all ENV var are global) will be `ONLYOFFICE_WEBDRIVER_WRAPPER_ALLOW_HTTP_DOWNLOAD` - or maybe a little shorter if you are more creative than me :)

And after that inside your project that uses this library call something
```
ENV['ONLYOFFICE_WEBDRIVER_WRAPPER_ALLOW_HTTP_DOWNLOAD' = ENV['PortalIP']`
```
so you will need to set in only once...

So @l8556  I advice to rename this ENV in this lib